resources/quality/tevo_blackwidow/tevo_blackwidow_draft.inst.cfg

```diff
@@ -25,7 +25,6 @@ support_angle = 60
 support_enable = True
 support_interface_enable = True
 support_pattern = triangles
-support_roof_enable = True
```

Comment:
Hold on, why did you remove this? It has nothing to do with this feature. Was it accidentally removed?

Reply:
Ohhhhh, my mistake. I wanted to do it in master and I was in this branch by mistake.